### PR TITLE
Adding qos capability on the channel used by consumers.

### PR DESCRIPTION
**Pull request by NiR-**

### CODE REVIEW

**Comment by mojoLyon** [comment]
:+1:

**Comment by mikaelrandy** [comment]
:+1: for the functionality, but the change on signature of the public method `AmqpBundle/Factory/ConsumerFactory:get()` could be considered as a BC Break.

Is this release should be a new major release ?

**Comment by omansour** [comment]
ping @fabdsp what you think ?

**Comment by mguillermin** [comment]
:+1: 

IMO if we want to follow semver, this should be new a major release (even if the `ConsumerFactory::get()` is probably never called directly).

**Comment by NiR-** [comment]
I have moved the `$qosOptions` at the end of the method signature. In that way, it will not make BC break :)

**Comment by omansour** [comment]
sounds :+1:

File: src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php [hunk 1]
```diff
@@ -134,6 +134,7 @@ protected function loadConsumers(ContainerBuilder $container, array $config)
                     new Reference(sprintf('m6_web_amqp.connection.%s', $consumer['connection'])),
                     $consumer['exchange_options'],
                     $consumer['queue_options'],
+                    $consumer['qos_options'],
```
Comment (mguillermin) [inline src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php]: `$consumer['qos_options']` should be after `$lazy` if you look at the definition of `ConsumerFactory::get()`

Comment (NiR-) [inline src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php]: Whoops, just forgot to update it.

**Comment by fabdsp** [comment]
:+1:

**Comment by mguillermin** [comment]
:+1:

**Pull request merged**